### ISSUE TITLE
Update 2020-01-22

### ISSUE THREAD

**Issue by theo-armour** [GreenBuildingXML/GreenBuildingXML.github.io#24]
Wok continued on developing a Viewer for the Ladybug Tools Honeybee format

[Read Me]( https://www.ladybug.tools/honeybee-viewer/ )

[Honeybee Viewer 3D]( https://www.ladybug.tools/honeybee-viewer/viewer-3d/ )

The relevant connection with the Spider gbXML viewers is that this viewer uses a nifty new parser to translate 3D vertices to Three,js meshes.

An early, broken version of the gbXML viewer is viewable here:

[Spider gbXML Viewer Dev]( https://www.ladybug.tools/spider-gbxml-tools/spider-gbxml-viewer/vx-2020-01-20/gbx-gbxml/gbx-threejs-basic-2020-01-21.html )

The big issue with the new viewer  is that shading is not working, but the basic parser is smaller and faster than ever. 

At the same time I have been refactoring all the underlying modules for use with the Viewer and other scripts - and these are also a work-in-progress:

https://pushme-pullyou.github.io/templates-01/threejs-hamburger/v-2020-01-17/threejs-hamburger-basic.html

The bottom line is there's a lot of work-in-progress and not that much to crow about.

***

Posted on the Ladybug Tools Discourse forum:

https://discourse.ladybug.tools/t/tip-fastest-way-to-view-a-gbxml-file/8172
